sw/device/lib/testing/_index.md

```diff
@@ -5,17 +5,30 @@ title: "Chip-Level Test Libraries"
 # Overview
 
 This subtree contains _test libary code_ that could aid in the writing of [chip-level tests]({{< relref "sw/device/tests/index.md" >}}).
+Test library code consists of two components:
+1. `testutils` libraries, and
+2. the [on-device test framework]({{< relref "sw/device/lib/testing/test_framework/index.md" >}}).
 
-# Organization and Style Guide
+Functions in `testutils` libraries are designed to wrap several DIF invocations that are commonly used together across many chip-level tests.
+They are _not_ designed to wrap a single DIF call.
 
-## File Naming Conventions
-- Test libary code will be named: **{IP or functionality name}_testutils.{h,c}**
+The [on-device test framework]({{< relref "sw/device/lib/testing/test_framework/index.md" >}}) provides a generic platform for writing chip-level tests.
 
-## Subfoldering Rules
-- [on-device test framework]({{< relref "sw/device/lib/testing/test_framework/index.md" >}})
-code will live in: **sw/device/lib/testing/test\_framework**.
-- Remaining test library code will **_not_** be subfoldered.
+# Style Guide
 
-# Documentation Index
+- All `testutils` libraries should be placed in `sw/device/lib/testing/*`
+- The [on-device test framework]({{< relref "sw/device/lib/testing/test_framework/index.md" >}})
+code will live in: sw/device/lib/testing/test\_framework.
+- `testutils` libraries will be named: {IP or functionality name}_testutils.{h,c}
+- All `testutils` function names should take on the following format: `<IP or functionality name>_testutils_<function name>()`.
+  This corresponds to the format: `<filename>_<function name>()`.
+- There is no strict return typing required, though ideally most `testutils` functions should return **void** or **bool*hjzz*.
```

Comment:
is this **bool*hjzz* intended?

Reply:
Oops. No, this was because Tim cannot type :rofl:. Good catch!

Fixed.